circle.yml

```diff
@@ -16,6 +16,9 @@ dependencies:
 test:
   override:
     - reaction test
+    - npm install -g jest
+    - npm update
+    - jest
```

Comment:
This is unnecessary because you can now do `npm test` and it will run the local copy of `jest` installed with reaction. (If the above code installs a different version of jest than you expect, then unforeseen issues may occur)